cpp/src/games/stochastic_nim/main/UnitTests.cpp

```diff
@@ -14,6 +15,99 @@ using Types = Game::Types;
 using SymmetryGroup = groups::TrivialGroup;
 using GameResults = core::WinShareResults<Game::Constants::kNumPlayers>;
 
+class PerfectPlayerTest : public testing::Test {
+ protected:
+  using PerfectPlayer = stochastic_nim::PerfectPlayer;
+  using PerfectStrategy = stochastic_nim::PerfectStrategy;
+  using State = PerfectPlayer::base_t::State;
+  using ActionMask = PerfectPlayer::ActionMask;
+
+ public:
+  PerfectPlayerTest() : strategy_(new PerfectStrategy()), player_(strategy_) {}
```

Comment:
You could have made `strategy_` a non-pointer, and then simply done here,

```
  PerfectPlayerTest() : player_(&strategy_) {}
```

And remove the destructor.